Cargo.lock

```diff
@@ -3368,4 +3368,4 @@ dependencies = [
  "sha3",
  "thiserror",
  "zeroize",
-]
+]
```

Comment:
Why do we have this change?

Reply:
Oh, I had to revert changes here. I'll add the EOF line